mitxgraders/listgrader.py

```diff
@@ -324,10 +324,6 @@ def validate_grouping(self):
                   "or a list of subgraders"
             raise ConfigError(msg)
 
-        # Unordered, we must have a single ListGrader subgrader
```

Comment:
Redundant error checking; this is tested for previously, as it applies regardless of whether or not grouping is used.